app/views/articles/_sidebar_additional.html.erb

```diff
@@ -7,8 +7,12 @@
           Join <%= ApplicationConfig["COMMUNITY_NAME"] %>
         </header>
         <div class="widget-body">
-          <a href="/users/auth/twitter?callback_url=<%= ApplicationConfig["APP_PROTOCOL"] %><%= ApplicationConfig["APP_DOMAIN"] %>/users/auth/twitter/callback" class="cta cta-button login-cta-button" data-no-instant>Sign In With Twitter</a>
-          <a href="/users/auth/github?state=navbar_basic" class="cta cta-button login-cta-button" data-no-instant>Sign In With GitHub</a>
+          <% if SiteConfig.auth_allowed?("twitter") %>
```

Comment:
Small nitpick, but is there a convenient place to constantize these strings? Are they from OmniAuth?